includes/core/server/request/class-create-and-confirm-setup-intention.php

```diff
@@ -79,7 +79,7 @@ public function set_metadata( array $metadata ) {
 	 * @throws Invalid_Request_Parameter_Exception
 	 */
 	public function set_payment_method( string $payment_method_id ) {
-		$this->validate_stripe_id( $payment_method_id, [ 'pm', 'src' ] );
+		$this->validate_stripe_id( $payment_method_id, [ 'pm', 'src', 'card' ] );
```

Comment:
@Automattic/gamma - it sounds like this request class `Create_And_Confirm_Setup_Intention` does not have tests under `tests/unit/core/server/request`. I think we will need to add tests for it.